src/javascripts/ng-admin/Main/config/routing.js

```diff
@@ -16,17 +16,21 @@ function entryConstructorProvider() {
 
 function routing($stateProvider, $urlRouterProvider) {
 
-    $stateProvider.state('main', {
+    $stateProvider.state('ng-admin', {
         abstract: true,
-        controller: 'AppController',
-        controllerAs: 'appController',
-        templateProvider: ['NgAdminConfiguration', function(Configuration) {
-            return Configuration().layout() || layoutTemplate;
-        }]
+        views: {
+            'ng-admin': {
+                controller: 'AppController',
+                controllerAs: 'appController',
+                templateProvider: ['NgAdminConfiguration', function(Configuration) {
+                    return Configuration().layout() || layoutTemplate;
+                }]
+            }
+        }
     });
 
     $stateProvider.state('dashboard', {
```

Comment:
Note that we can also use dotted notation for parent views: `ng-admin.dashboard` for instance.